cmd/subnetcmd/join.go

```diff
@@ -73,6 +74,8 @@ func init() {
 		wd,
 		home,
 		filepath.Join(home, constants.AvalancheGoRepoName),
+		filepath.Join(home, "go", "src", "github.com",
```

Comment:
Can we make these constants?

Reply:
We could make constants for each part ("go", "src", etc.) individually. Which seems cumbersome.

But we can't make it a constant due to the filepath.Join() construct. 
I made it a `var` local to this file (and package) for now